cli/test/test_cmdversion.py

```diff
@@ -0,0 +1,12 @@
+from click.testing import CliRunner
+import popper.commands.cmd_version as version
+import unittest
+from popper_test import PopperTest
+
+
+class TestCommandVersion(unittest.TestCase, PopperTest):
+
+    def test_version(self):
+        runner = CliRunner()
+        result = runner.invoke(version.cli)
+        assert result.exit_code == 0
```

Comment:
we can check that the version string returned is the same as [`popper.__init__.py`](https://github.com/systemslab/popper/blob/master/cli/popper/__init__.py)

Reply:
Ok, I can add that here.